test/automation-aggregator.ts

```diff
@@ -626,176 +639,13 @@ describe('AutomationAggregatorBot', async () => {
             )
             expect(status).to.equal(true, 'approval does not exist for AutomationBot')
         })
-    })
-    describe('replaceGroupTrigger', async () => {
-        async function executeTrigger(triggerId: number, targetRatio: BigNumber, triggerData: BytesLike) {
```

Comment:
We need test prooving that triggers groups registered this way are still executable, able to operate on vaults.